packages/nimble-components/src/anchor-menu-item/tests/anchor-menu-item.spec.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable max-classes-per-file */
 import { customElement, html, ref } from '@microsoft/fast-element';
 import { MenuItem as FoundationMenuItem } from '@microsoft/fast-foundation';
+import { parameterizeNamedList } from '@ni/jasmine-parameterized/dist/esm/parameterized';
```

Comment:
> Don't expose the parameterize helper function (i.e. don't export it or put it in the readme) and rename parameterizeNamedList to parameterizeSpec

I've renamed `parameterizeNamedList` to `parameterizeSpec`. I also no longer am exporting `parameterize` or including it in the readme. However, my question is: **Should we remove `parameterize` altogether?** It isn't used by anything.**

> Make it so the import is top-level in the package, i.e. import { parameterizeSpec } from '@ni/jasmine-parameterized';. This should be doable with conditional exports for import and require: https://nodejs.org/api/packages.html#conditional-exports

Done, but let me know if there is something I should've done differently.

> We create the cjs build and we should test it in node

Done. I also updated `karma.conf.json` to only run tests on the `esm` build, but **should `karma.conf.json` still be configured to run for both `esm` and `commonjs`?**

Reply:
> Should we remove parameterize altogether? It isn't used by anything.**

It is used in the parameterizeSpec implementation. Don't think I follow what is being proposed.

https://github.com/ni/nimble/blob/7be0f7ccab419e951923c37eb26fba56c7e0420e/packages/jasmine-parameterized/src/parameterized.ts#L125

> Done. I also updated karma.conf.json to only run tests on the esm build, but should karma.conf.json still be configured to run for both esm and commonjs?

Was mulling over it a bit and realized that the browser usage was being pushed through webpack, so it was really testing the webpack config and the node usage wasn't relying on the cjs import as the test file itself was using the es6 `import` statement to reference the dependency instead of the cjs format `require` statement.

Modern versions of browsers and node support es6 modules so I simplified and removed the cjs parts of the build. I switched the TS config to use the newer "Node16" for the module format (which is just es6 modules as supported by node).

So now we are actually using esm js files and I got [esm tests to work in karma](https://github.com/karma-runner/karma/issues/3335#issue-464509910) .

I also read the [npm entrypoints](https://nodejs.org/api/packages.html#package-entry-points) and [typescript npm entrypoint](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-7.html#packagejson-exports-imports-and-self-referencing) docs and modified the format of the entrypoints to match that. For some reason I needed the `Fall-back for older versions of TypeScript` that I couldn't figure out but guess it's not a big deal.